ts/twitter/src/lib/components/sidebar.tsx

```diff
@@ -1,24 +1,24 @@
 "use client";
 
-import React from "react";
+import { BellIcon, EmailIcon, SearchIcon } from "@chakra-ui/icons";
 import {
+  Box,
+  Flex,
   IconButton,
+  Spacer,
+  Text,
   Tooltip,
   VStack,
-  Flex,
-  Box,
-  Text,
-  Spacer,
 } from "@chakra-ui/react";
-import { SearchIcon, BellIcon, EmailIcon } from "@chakra-ui/icons";
-import { PiHouseFill } from "react-icons/pi";
-import { FaTwitter, FaUser } from "react-icons/fa";
-import { BsSlashSquare, BsPeople } from "react-icons/bs";
-import { CiCircleMore } from "react-icons/ci";
 import Link from "next/link";
+import type React from "react";
+import { BsPeople, BsSlashSquare } from "react-icons/bs";
+import { CiCircleMore } from "react-icons/ci";
+import { FaTwitter, FaUser } from "react-icons/fa";
+import { PiHouseFill } from "react-icons/pi";
 import IconButtonWithLink from "./icon-button-with-link";
-import SignInbutton from "./sign-in-button";
 import { SideBarPostButton } from "./sidebar-post-button";
+import SignInbutton from "./sign-in-button";
```

Comment:
Since the code has been changed to use a named export instead of a default export, update the export/import syntax like this:
```suggestion
import { SignInbutton } from "./sign-in-button";
```

Reply:
Thank you for pointing that out. For some reason, VSCode didn't flag the error.  
In addition to fixing it, I also noticed that some other components were using default exports, so I updated them as well.